### PR TITLE
refactor(console): update search placeholder

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

- Update search placeholder
- Add a search placeholder for the permissions table
- Add a search placeholder for the user roles table
- Support configuring placeholder in the Search component and increase the search component's width to 306px according to design file.

## Testing

image
image
image

### CODE REVIEW

File: packages/console/src/components/Search/index.module.scss [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
   align-items: center;
 
-  :not(:first-child) {
+  >:not(:first-child) {
     margin-left: _.unit(2);
   }
 
   .searchInput {
-    width: 286px;
+    width: 306px;
```
Comment (charIeszhao) [inline packages/console/src/components/Search/index.module.scss:12]: Why do we use a fixed width for the search component?

Comment (xiaoyijun) [reply packages/console/src/components/Search/index.module.scss:12]: The original search component needs longer to display the placeholder, and the `TextInput` component does not support automatic width adjustment according to the placeholder content.
So follow the original implementation this time.

image

File: packages/console/src/components/Search/index.tsx [hunk 2]
```diff
@@ -39,7 +46,7 @@ const Search = ({ defaultValue = '', isClearable = false, onSearch, onClearSearc
         <TextInput
           value={inputValue}
           icon={<SearchIcon className={styles.searchIcon} />}
-          placeholder={t('general.search_placeholder')}
+          placeholder={placeholder ?? t('general.search_placeholder')}
```
Comment (simeng-li) [inline packages/console/src/components/Search/index.tsx:49]: nit: The usage seems unusual.  Should let the upper scope control the placeholder content? What if some page meant to keep the placeholder empty have to pass an empty string?

**Review by simeng-li — COMMENTED** [review]
overall LGTM

File: packages/console/src/components/Search/index.module.scss (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/components/Search/index.module.scss]: Finally, I found that the width settings should be specified separately.
Already updated.

File: packages/console/src/components/Search/index.tsx (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/components/Search/index.tsx]: Cool, updated.

**Review by charIeszhao — APPROVED** [review]

**Pull request merged**